NSS-Day-Cohort-49/helloworld-git-clone-crew
# Lrc sweden

## Steps to test

1. Pull the code down 
2. see that the sweden html shows up in editor 
3. see that the navbar html show up in editor
4. serve the code up 
5. see that all of the code shows up in the browser
